### DISCUSSION TITLE
Loadiiiiiing

### DISCUSSION THREAD

**Discussion by boscoeroscoe** [IceHacks/SurvivCheatInjector#776] [General]
I install the file correctly yet it loads for years without loading
I left it loading for 1 hour and it didn't load.

**Comment by boscoeroscoe** [comment]
plz help

**Reply by ghost** [reply]
sus

**Reply by AsiaGuy** [reply]
@boscoeroscoe your pc bad lmao, it take at least 1-15min to load

**Comment by IceHacks** [comment]
What are your system specs? Also, I don't recommend leaving it running for that long. Try reinstalling it. Once it loads, make sure that the performance options are OFF.
